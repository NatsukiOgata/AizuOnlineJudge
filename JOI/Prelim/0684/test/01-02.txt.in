95 90
57 95 52 52 72 54 11 31 69 3 35 81 56 34 95 45 45 66 57 81 20 39 81 3 17 4 41 52 92 34 95 54 12 31 84 42 68 45 54 34 70 52 92 52 66 99 56 80 42 50 14 95 52 6 13 13 2 68 52 31 66 66 31 68 72 6 99 4 35 72 42 70 99 81 35 81 80 8 66 91 39 52 14 39 15 72 34 4 68 41 41 95 12 91 15
69 12 34 81 80 57 35 84 34 12 69 39 42 50 6 57 3 3 12 99 69 81 31 52 34 11 6 39 47 30 30 6 81 20 81 8 47 41 70 39 52 30 95 84 6 4 6 95 30 72 30 13 13 52 72 42 47 81 35 31 81 45 70 72 11 17 35 95 30 39 45 70 39 17 47 56 13 81 4 45 6 2 2 3 81 6 70 57 42 54
