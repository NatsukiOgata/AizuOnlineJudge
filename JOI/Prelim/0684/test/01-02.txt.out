2
3
4
6
8
11
12
13
17
20
31
34
35
39
41
42
45
50
52
54
56
57
69
70
72
80
81
84
95
99
